src/controllers/comments.controller.js

```diff
@@ -26,17 +26,7 @@ const CommentsController = {
     sendResult(res, 201, 'Comment Created', newComment);
   },
   viewComment: async (req, res) => {
```

Comment:
the same here with our team function naming convention and consistence in fact the whole folder is off

Reply:
this piece of code was already there